PowerEditor/src/NppCommands.cpp

```diff
@@ -3555,6 +3555,77 @@ void Notepad_plus::command(int id)
 		}
 		break;
 
+		case IDM_WINDOW_SORT_FN_ASC :
+		{
+			WindowsDlg _windowsDlg;

```

Comment:
Underscore at the beginning is usually used only for member variables.
https://github.com/notepad-plus-plus/notepad-plus-plus/blob/master/CONTRIBUTING.md#member-variables